doc/getting_started.rst

```diff
@@ -20,17 +18,200 @@ by the `FreeSurfer`_ software.
 
 .. _FreeSurfer: http://surfer.nmr.mgh.harvard.edu
 
-Downloading and installing the Unix commands
---------------------------------------------
-
 .. note::
 
-    If you are working at the Martinos Center see :ref:`setup_martinos`
-    for instructions to work with MNE and to access the Neuromag software.
+    If you are at the Martinos Center, please see this section :ref:`inside_martinos`.
+
+
+Install Python and MNE-Python
+-----------------------------
+
+To use MNE-Python, you need two things:
+
+1. :ref:`A working Python interpreter and dependencies<install_interpreter>`
+
+2. :ref:`MNE-Python installed to the Python distribution <install_mne_python>`
+
+
+.. _install_interpreter:
+
+Install a Python interpreter and dependencies
+#############################################
+
+There are multiple options available for getting a suitable Python interpreter
+running on your system. However, for a fast and up to date scientific Python
+environment that resolves all dependencies, we highly recommend following
+installation instructions for the Anaconda Python distribution. Go here
+to download, and follow the installation instructions:
+
+https://www.continuum.io/downloads
+
+Anaconda is free for academic purposes. You can also
+`sign up for a subscription free for academic use <https://www.continuum.io/anaconda-academic-subscriptions-available>`_
+that allows installation of the ``accelerate`` package to speed up
+comptutations up to 10x by using
+`Intel MKL <https://software.intel.com/en-us/intel-mkl>`_ as:
+
+.. code-block:: bash
+
+    $ conda install accelerate
+
+Once Anaconda works properly, you can do this to resolve mne-python dependencies:
+
+.. code-block:: bash
+
+    $ conda install scipy matplotlib scikit-learn
+
+To test that everything works properly, open up IPython:
+
+.. code-block:: bash
+
+    $ ipython --matplotlib=qt
+
+Now that you have a working Python environment you can install MNE-Python.
+
+.. _install_mne_python:
+
+Install mne-python
+##################
+
+Most users should start with the "stable" version of mne-python, which can
+be installed this way:
+
+.. code-block:: bash
+
+    $ pip install mne --upgrade
+
+For the newest features (and potentially more bugs), you can instead install
+the development version by:
+
+.. code-block:: bash
+
+    $ pip install -e git+https://github.com/mne-tools/mne-python#egg=mne-dev
+
+If you plan to contribute to the project or want to be able to update
+your MNE-Python version very frequently (weekly or monthly), please follow
+the :ref:`git instructions <contributing>`.
+
+Check your installation
+#######################
+
+To check that everything went fine, in ipython, type::
+
+    >>> import mne
+
+If you get a new prompt with no error messages, you should be good to go!
+
+Familiarize yourself with Python
```

Comment:
I'm not sure if this is useful, but a lot of people find this instructive: https://learnxinyminutes.com/docs/python/